### PR TITLE
Remove linefeed from og:description

### LINKED ISSUE #220: og:descriptionに改行が入ることがある
カテゴリーのページで確認。
本文に改行が入っていても、og:descriptionに入れるときに削除したほうがよさそう。

### PR DESCRIPTION
**Pull request by myokoym**
Close #220

### CODE REVIEW

File: app/decorators/post_decorator.rb [hunk 1]
```diff
@@ -8,7 +8,7 @@ def to_og_params
       locale: 'ja_JP',
       type: 'article',
       title: title,
-      description: render_markdown(body),
+      description: render_markdown(body)&.gsub("\n", ''),
```
Comment (tricknotes) [inline app/decorators/post_decorator.rb]: I think `body` may be `nil`.
By the way using `to_s` might be meaningful.

**Comment by myokoym** [comment]
TODO:
- Add tests.
- Support `nil` for `body`.
- Use `to_s` instead of `&.` if needed.

File: app/decorators/post_decorator.rb (hunk 1 above)
Comment (myokoym) [inline app/decorators/post_decorator.rb]: Thanks! I added TODO.

Comment (cosmo0920) [inline app/decorators/post_decorator.rb]: > By the way using to_s might be meaningful.

:+1: 

It always exists an isomorphism between ""(empty string) and `nil`. (a.k.a. `nil` can convert "" in String type.)

Comment (mtsmfm) [inline app/decorators/post_decorator.rb]: By the way, posts always should have some contents.
So how about adding not null constraint to `posts.body`?

Comment (tricknotes) [inline app/decorators/post_decorator.rb]: I totally agree.

Comment (cosmo0920) [inline app/decorators/post_decorator.rb]: > adding not null constraint to posts.body?

It sounds good for me.

**Comment by tricknotes** [comment]
looks good :sparkles:

**Comment by cosmo0920** [comment]
Awesome! :+1:

**Comment by mtsmfm** [comment]
:shipit:

**Pull request merged**